### ISSUE TITLE
CT: append-contrib.js parsing errors

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/community#19]
I discovered this after I did a pull-all, then tried to `grunt lint-everything`.

CTQ is also reporting it:

```
community:


/Users/cmalley/PhET/GitHub/community/scripts/append-contrib.js
  11:1  error  Parsing error: The keyword 'import' is reserved

/Users/cmalley/PhET/GitHub/community/scripts/append-to-file.js
  9:1  error  Parsing error: The keyword 'import' is reserved

/Users/cmalley/PhET/GitHub/community/scripts/copy-pr-template.js
  11:1  error  Parsing error: The keyword 'import' is reserved

/Users/cmalley/PhET/GitHub/community/scripts/core-common-repos.js
  9:1  error  Parsing error: The keyword 'const' is reserved

/Users/cmalley/PhET/GitHub/community/scripts/is-in-phet-root.js
  9:1  error  Parsing error: The keyword 'import' is reserved

✖ 5 problems (5 errors, 0 warnings)
```

It looks like append-contrib.js was created yesterday (3/10/23) by @liammulh.

**Comment by pixelzoom** [comment]
So that I can continue working, I added this workaround (suggested by @samreid) to lint.js in the above commit:

```ts
const EXCLUDE_REPOS = [

  //TODO workaround for https://github.com/phetsims/community/issues/19, remove this when resolved
  'community'
];
```

@liammulh please make sure this workaround gets removed when you resolve this issue.

**Comment by samreid** [comment]
This will be addressed by #21, @liammulh will close when #21 is complete.

**Comment by liammulh** [comment]
The work done in https://github.com/phetsims/community/issues/21 should fix this. Closing.
